src/js/administration/api.ts

```diff
@@ -0,0 +1,66 @@
+import { Request } from "../app/request";
+/**
```

Comment:
Add module docstring if you are going to do it elsewhere.
```suggestion
import { Request } from "../app/request";

/**
```